Source/Cake.CakeMail/Email/CakeMailEmailProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
```

Comment:
Oh, I see. I haven't looked at that code in a while 🙂  In which case, I would suggest refactoring this to be simpler, and I can do the same in Cake.Gitter.